javascript/builtins/intl/Collator.json

```
@@ -77,9 +84,16 @@
                 "ie": {
                   "version_added": "11"
                 },
-                "nodejs": {
-                  "version_added": null
-                },
+                "nodejs": [
+                  {
+                    "version_added": "13.0.0"
+                  },
+                  {
+                    "version_added": "0.12",
+                    "partial_implementation": true,
+                    "notes": "Before version 13.0.0, only the locale data for <code>en-US</code> is available by default. When other locales are specified, the <code>Collator</code> instance silently falls back to <code>en-US</code>. In order to make full ICU (locale) data available for versions prior to 13, see docs on the <code>--with-full-icu=</code> flag and how to provide the data."
```

For the full-length notes, I got to thinking that it would be nice to link to the docs. This revealed that our original note is actually wrong about the build flag. Since you said you had a script to rewrite the notes, I thought it might be nice to update this now (minus the inconsistent notes you mentioned). This does a little editing too, while I'm at it.

```suggestion
                    "notes": "Before version 13.0.0, only the locale data for <code>en-US</code> is available by default. When other locales are specified, the <code>Collator</code> instance silently falls back to <code>en-US</code>. To make full ICU (locale) data available for versions prior to 13, see <a href="https://nodejs.org/docs/latest/api/intl.html#intl_options_for_building_node_js">Node.js documentation on the <code>--with-intl</code> option</a> and how to provide the data."
```

---

Great suggestion, I hadn't thought to add links to the Node docs.

> This revealed that our original note is actually wrong about the build flag.

Oops! I didn't notice that in all the time I've spent looking at those messages.